springerlink.md

```diff
@@ -975,16 +1111,102 @@ Images of product covers (e.g. journals or books displayed on the homepage of li
 Third party advertising scripts may inject images containing images of text.
 </td>
 </tr>
+<tr id="reflow" valign="top">
+<td><a href="https://www.w3.org/TR/WCAG21/#reflow"><strong>1.4.10 Reflow</strong></a> (Level AA 2.1 only)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.1.4.10 (Web)</li>
+<li>10.1.4.10 (non-web document)</li>
+<li>11.1.4.10.1 (Open Functionality Software)</li>
+<li>11.1.4.10.2 (Closed Software) – Does not apply</li>
+<li>11.8.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508 - does not apply</em>
+</td>
+<td>Supports</td>
+<td>All parts of this product are responsive. Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions per the specifications in this criterion.</td>
+</tr>
+<tr id="non-text-contrast" valign="top">
+<td><a href="https://www.w3.org/TR/WCAG21/#non-text-contrast"><strong>1.4.11 Non-text Contrast</strong></a> (Level AA 2.1 only)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.1.4.11 (Web)</li>
+<li>10.1.4.11 (non-web document)</li>
+<li>11.1.4.11 (Open Functionality Software)</li>
+<li>11.1.4.11 (Closed Software) – Does not apply</li>
+<li>11.8.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508 - does not apply</em>
+</td>
+<td>Partially Supports</td>
+<td>
+The SpringerLink website contains content that is rendered in two separate designs. We are in the process of transferring all content from the older design to the newer design. 
```

Comment:
Shared renderer on article pages will kind of make it the third design 😅maybe this first sentence about two designs can be taken out and just keep the distinction between older and newer designs?

Reply:
Oh yeah! Though the "old" vs "new" designs are really referring to the old version of Link with the gradients and the tiny writing vs the modern white and blue style. I can't easily tell the difference between the shared renderer version and the non-shared renderer version - they both look like they're using the same design to me, even if the HTML is different and the layout is slightly changed. Users would probably feel the same way. 

There's no change to the accessibility between the shared renderer version and the previous version, and you're about to hit the switch to send 100% of traffic to the shared renderer anyway, so I'm leaning towards thinking this is probably ok as-is. What do you reckon?